test/cpp/interop/interop_server.cc

```diff
@@ -360,6 +386,9 @@ void grpc::testing::interop::RunServer(
   if (absl::GetFlag(FLAGS_max_send_message_size) >= 0) {
     builder.SetMaxSendMessageSize(absl::GetFlag(FLAGS_max_send_message_size));
   }
+  auto metric_recorder = experimental::ServerMetricRecorder::Create();
```

Comment:
In this particular case, we don't need a `ServerMetricRecorder`, so I don't think it's necessary to create this.  You can just pass null to `EnableCallMetricRecording()`.

Reply:
Done.